steps/nvm/1.0.0/step.yml

```diff
@@ -0,0 +1,46 @@
+title: Node Version Manager (NVM)
+summary: Install NVM and select Node version
+description: |
+  This step will install a specific node.js version using NVM, and will make it available for all your other steps.
+
+  NVM (Node Version Manager) is a simple bash script to manage multiple active node.js versions.
+  See [here](https://github.com/creationix/nvm) for more details.
+website: https://github.com/almouro/bitrise-nvm-step
+source_code_url: https://github.com/almouro/bitrise-nvm-step
+support_url: https://github.com/almouro/bitrise-nvm-step/issues
+published_at: 2016-07-24T19:29:12.711068565+02:00
+source:
+  git: https://github.com/Almouro/bitrise-nvm-step.git
+  commit: 215a76f4a17401e5cb42d079c7321e37780f7c47
+host_os_tags:
+- osx
+- ubuntu
+project_type_tags:
+- react-native
```

Comment:
we don't have react-native as a project type yet - this means that this step won't show up for any project by default, until you turn off the "filter by project type" toggle in the step list. This step I think should be general enough, so you should simply remove the `project_type_tags` list - that means that the step can work with any project type. Of course, if that's not the case, you can keep it.